### PR TITLE
fix some flake8 linting error

### LINKED ISSUE #428: Pre-commit finding some issue
Some flake8 error made their way into the code. I realized that when trying to configure ruff.

```
$ pre-commit run --all
fix end of files.........................................................Passed
check for case conflicts.................................................Passed
trim trailing whitespace.................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

examples/strava-oauth/server.py:2:1: F401 'logging' imported but unused
examples/strava-oauth/server.py:4:1: F401 'flask.jsonify' imported but unused
examples/strava-oauth/server.py:4:1: F401 'flask.redirect' imported but unused
stravalib/__init__.py:1:1: F401 'stravalib.client.Client' imported but unused
stravalib/tests/integration/strava_api_stub.py:132:25: F541 f-string is missing placeholders

black....................................................................Passed
isort....................................................................Passed
```

I don't quite know how this is possible, but I'd assume that it's from before it was well integrated in CI. I don't think we should worry too much about it. 

Incoming PR to fix the errors.

### PR DESCRIPTION
**Pull request by enadeau**
closes #428

## Description

Fixing some linting error. Mostly unused import. See the issue for details.

## Type of change

Select the statement best describes this pull request.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This is a documentation update
- [ ] Other (please describe)

## Does your PR include tests

If you are fixing a bug or adding a feature, we appreciate (but do not require)
tests to support whatever fix of feature you're implementing.

- [ ] Yes
- [ ] No, i'd like some help with tests
- [x] This change doesn't require tests

## Did you include your contribution to the change log?

- [ ] Yes, `changelog.md` is up-to-date.

No changelog entry necessary here I think.

### CODE REVIEW

File: stravalib/__init__.py [hunk 1]
```diff
@@ -6,3 +6,6 @@
     from ._version_generated import __version__
 except ImportError:
     __version__ = "unreleased"
+
+
+__all__ = ["Client"]
```
Comment (lwasser) [inline stravalib/__init__.py:11]: nice. i saw this was the fix from a flake8 maintainer for the flake8 unused import issue !

**Comment by lwasser** [comment]
actually @enadeau i made this fix in the PR i just merged. i ran all pre-commit hooks locally and tried to fix everything so we likely don't need this pr. i know it's easy to miss because that PR i just merged fixed small things in like 50+ files 😆 but maybe rebase from main and see if you still catch issues. as in theory i should have cleaned everything up. But you know... in practice... maybe i missed things!

File: stravalib/__init__.py (hunk 1 above)
Comment (enadeau) [reply stravalib/__init__.py]: Yes and it's also to practice to define explicitly what you are reexporting. Some tool will complain about using stuff that is not explicitly reexported

**Comment by enadeau** [comment]
Just rebased on main. If you approve I'll merge

**Comment by enadeau** [comment]
Ah yeah I see we've made the same change just closing everything then!

File: stravalib/__init__.py (hunk 1 above)
Comment (lwasser) [reply stravalib/__init__.py]: oh - can you explain what you mean by re-exporting? i understand the import issue now.
And thanks for opening this. i know we have made a LOT of collective changes in the past week (but all good ones!!)

Comment (enadeau) [reply stravalib/__init__.py]: It means you explicitly make it part of your api for this module. See maybe here for details https://mypy.readthedocs.io/en/stable/command_line.html?highlight=--no-explicit-reexport#cmdoption-mypy-no-implicit-reexport

**Pull request closed without merging**